apps/TideWall/main.cpp

```diff
@@ -85,13 +85,14 @@ int main(int argc, char* argv[])
         qInstallMessageHandler(qtMessageLogger);
 
         MPIChannelPtr localChannel(new MPIChannel(*worldChannel, 1, rank));
+        MPIChannelPtr localChannel2(new MPIChannel(*worldChannel, 1, rank));
```

Comment:
You're right, I'm also not happy with those names. I tried to keep them generic because they appear in all 3 applications (master, wall, forker) but in the end I never remember what they are used for. I'll try to improve that.